.env

```diff
@@ -0,0 +1,3 @@
+REACT_APP_HOST=http://localhost:3000
+REACT_APP_OPEN_WEATHER_KEY_API=51a234de54ee35fad3d91b344c8e0b57
+REACT_APP_OPEN_WEATHER_URL=http://api.openweathermap.org/
```

Comment:
No. The names of these files you are going to upload are: 
`.env.example`
`.env.production`
`.env.developent`

`.env` files are similar to `.env.local`. That is why those files are ignored by devs.